### ISSUE TITLE
Deployed version doesn't seem to be the same version in the `main` branch

### ISSUE THREAD

**Issue by gearsandcode** [sully-justsully/colorgrid#3]
On https://colorgrid.design/:
<img width="393" alt="Image" src="https://github.com/user-attachments/assets/eb21b5ea-041d-4665-928e-3619851886af" />

There's a View Color Contrast and the WCAG controls are checkboxes.

---

On the `main` branch View Color Contrast is missing and the WCAG controls are radios. Additionally, the WCAG controls don't seem to work.

<img width="396" alt="Image" src="https://github.com/user-attachments/assets/e0f04b01-8f80-4059-9cef-d01468e67528" />
